### PR TITLE
Sample Image using WDT to build a WLS Domain and Coherence

### PR DESCRIPTION
**Pull request by pfmackin**
This sample demonstrates how to create a WebLogic 12.2.1.3 domain with a Coherence cluster using WebLogic Deploy Tooling (WDT).  The sample also includes a Coherence proxy server that runs inside the domain, along with a proxy client to access a Coherence cache.

### CODE REVIEW

File: OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+#
+# You must insure that the build arguments align with the values in the model. 
+# The sample model replaces the attributes with tokens that are resolved from values in the
+# corresponding property file domain.properties. The container-scripts/setEnv.sh script
+# demonstrates parsing the variable file to build a string of --build-args that can
+# be passed on the docker build command.
+#
+# Pull base image
+# ---------------
+FROM container-registry.oracle.com/middleware/weblogic:12.2.1.3
```
Comment (mriccell) [inline OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile]: This should extend FROM oracle/weblogic:12.2.1.3-developer

File: OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+# Maintainer
+# ----------
+MAINTAINER Richard Killen <richard.killen@oracle.com>
```
Comment (mriccell) [inline OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile]: Maintainer Paul Mackin

File: OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/README.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+It is the responsibility of the user to manage this volume, and the security.properties, in the container.
+
+**Note**: Oracle recommends that the `adminpass.properties`, `adminuser.properties`, and `security.properties` files 
+be deleted or secured after the image is built and the WebLogic Server is started so that the user name 
+and password are not inadvertently exposed.
+
+## How to Build and Run
+
+**NOTE:** The image is based on the following WebLogic Server image: 
+    `container-registry.oracle.com/middleware/weblogic:12.2.1.3`
```
Comment (mriccell) [inline OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/README.md]: build image under OracleWebLogic/dockerfiles/12.2.1.3 or pull image from OCR and run docker target

File: OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile (hunk 1 above)
Comment (pfmackin) [reply OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile]: Ok, see my note is slack.

File: OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/README.md (hunk 3 above)
Comment (pfmackin) [reply OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/README.md]: fixed

File: OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile (hunk 2 above)
Comment (pfmackin) [reply OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile]: fixed

File: OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile (hunk 1 above)
Comment (pfmackin) [reply OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile]: Fixed

**Pull request merged**